docs/getting-started.md

```diff
@@ -29,3 +29,79 @@ The author of this project strongly believes descriptors will be a big part of t
 technology and tooling of Bitcoin evolves and changes (Schnorr signatures, Taproot, etc).
 
 To learn more, check out the specific [Descriptors section](/descriptors).
+
+The following code shows how to generate a random mnemonic, the extended (and deterministic) keys from that mnemonic and finally the descriptors from the extended private keys.
+
+To be able to run this code, the `bdk` dependency in `Cargo.toml` must be set as follows:
+
+```
+[dependencies]
+bdk = { version = "0.15.0", default-feature = false, features = ["compiler", "all-keys"] }
```

Comment:
I think you don't need `compiler` here for just the descriptor generation code.

Reply:
Thanks. Removed.